src/components/dashboard/FeedItems/ListEventItem.js

```diff
@@ -53,47 +59,65 @@ const InviteItem = ({ item, theme }) => (
+    fireEvent(GOTO_SPONSOR)
+    Linking.openURL(sponsoredLink).catch(e => log.error('Open news feed error', e))
+  }, [sponsoredLink])
+  return (
+    <View style={styles.rowContent}>
+      <FeedListItemLeftBorder style={styles.rowContentBorder} color={eventSettings.color} />
 
-    <View style={styles.newsContent}>
-      {item.data.picture && <Image source={{ uri: item.data.picture }} style={styles.newsPicture} />}
+      <View style={styles.newsContent}>
+        {item.data.picture && <Image source={{ uri: item.data.picture }} style={styles.newsPicture} />}
 
-      <View style={styles.innerRow}>
-        <View grow style={styles.mainContents}>
-          <View style={{ ...styles.transferInfo, height: undefined }} alignItems="flex-start">
-            <View style={[styles.mainInfo, item.type === 'claiming' && styles.claimingCardFeedText]}>
-              <EventCounterParty
-                style={styles.feedItem}
-                feedItem={item}
-                subtitle
-                isSmallDevice={isMobile && getScreenWidth() < 353}
-                numberOfLines={2}
-              />
+        <View style={styles.innerRow}>
+          <View grow style={styles.mainContents}>
+            <View style={{ ...styles.transferInfo, height: undefined }} alignItems="flex-start">
+              <View style={[styles.mainInfo, item.type === 'claiming' && styles.claimingCardFeedText]}>
+                <EventCounterParty
+                  style={styles.feedItem}
+                  feedItem={item}
+                  subtitle
+                  isSmallDevice={isMobile && getScreenWidth() < 353}
+                  numberOfLines={2}
+                />
 
-              <Text
-                lineHeight={20}
-                numberOfLines={3}
-                color="gray80Percent"
-                fontSize={12}
-                textTransform="capitalize"
-                style={styles.newsMessage}
-              >
-                {get(item, 'data.message')}
+                <Text
+                  lineHeight={20}
+                  numberOfLines={3}
+                  color="gray80Percent"
+                  fontSize={12}
+                  textTransform="capitalize"
+                  style={styles.newsMessage}
+                >
+                  {get(item, 'data.message')}
+                </Text>
+              </View>
+            </View>
+            <View style={styles.dateContainer}>
+              <Text fontSize={10} color="gray80Percent" lineHeight={17}>
+                {getFormattedDateTime(item.date)}
               </Text>
+
+              {!!sponsoredLink && !!sponsoredLogo && (
+                <Pressable style={styles.sponsorContainer} onPress={onSponsorPress}>
+                  <Text fontSize={10} color="gray80Percent" lineHeight={17} textAlign="left">
+                    Sponsored by{' '}
+                  </Text>
+                  <SvgImage src={sponsoredLogo} height="15" width="47" />
```

Comment:
do we know that this will always be an svg?

Reply:
yes, this was discussed with @patpedrosa 
svg will be used as sponsor's logo to decrease amount of data has been transmitting

Reply:
Yes, it was planned